landscape.yml

```diff
@@ -74,10 +74,15 @@ landscape:
             name: Cadence Workflow
             description: Cadence is a distributed, scalable, durable, and highly available fault-oblivious stateful code platform.
             homepage_url: https://cadenceworkflow.io/
-            repo_url: https://github.com/uber/cadence
+            project: sandbox
+            repo_url: https://github.com/cadence-workflow/cadence
             logo: cadenceworkflow.svg
-            twitter: https://twitter.com/UberEng
-            crunchbase: https://www.crunchbase.com/organization/uber
+            twitter: https://twitter.com/CloudNativeFdn
```

Comment:
It's not that active but we have https://x.com/cadenceworkflow
We were thinking of being much more active after the CNCF move.

Reply:
We also have
https://www.linkedin.com/company/cadenceworkflow
and 
https://www.youtube.com/@cadenceworkflow2989

if we are listing all communication channels.